src/perception/tag_detector/tag_detector.processing.cpp

```diff
@@ -112,10 +113,18 @@ void FiducialsNode::imageCallback(sensor_msgs::ImageConstPtr const& msg) {
  * @param v     Y Pixel Position
  */
 std::optional<SE3> getFidInCamFromPixel(PointCloudPtr const& cloudPtr, size_t u, size_t v) {
+    if (u >= cloudPtr->width || v >= cloudPtr->height) {
```

Comment:
can this even happen?

Reply:
Yeah it is what caused the segfault observed at testing

Reply:
Another check I will be adding is if the point cloud and images are different dimensions, which should be a fatal exception.

Reply:
uh why does this happen?

Reply:
The ZED ROS wrapper has two parameters. One controls the resizing factor of the image and the other for the point cloud. Those need to be the same, but it is possible they aren't if the yaml is misconfigured.